docs/setup-vendor.md

```diff
@@ -4,6 +4,92 @@ Using Private Packagist for Vendors, your customers can install packages with Co
 
 Customers can use your packages in either an existing or a new Composer project.
 
+## Setting up a customer in Private Packagist
+
+You can manage your customers and grant them access to packages on the vendor page in Private Packagist.
+To create your first customer click on "Create Customer" and fill out the form on the next page.
+
+The customer's short name will be used in the Composer repository URL and the web UI URL where your customer sees packages
+they have access to and can download archives. Please be aware that changing the URL will change the web UI URL and
+the Composer repository URL and will therefore break existing Composer projects. The minimum accesible stability
+setting allows you to limit which versions of a package a customer has access to. More about that in the
+["Limit package access by minimum stability"](#limit-package-access-by-minimum-stability) section. You can also
+enable package source URLs for a customer. By default, Private Packagist removes source URLs for all packages
+assigned to a customer, e.g. for a git repository the URL to clone the repository, and customers are only able to install
+packages as archives. If your customer has access to your repositories and you want them to be able to install
+packages from source e.g. via git clone then you can enable the source code URL delivery to customers through Composer.
```

Comment:
Can you refactor this long paragraph into a list of the form options with an explanation each?

Reply:
done